app/helpers/application_helper/toolbar/x_vm_center.rb

```diff
@@ -290,6 +290,16 @@ class ApplicationHelper::Toolbar::XVmCenter < ApplicationHelper::Toolbar::Basic
           :url         => "native_console",
           :klass       => ApplicationHelper::Button::VmNativeConsole
         ),
+        button(
+          :vm_native_console,
+          'pficon pficon-screen fa-lg',
+          N_('Open a management console for this VM'),
+          N_('Management Console'),
+          :keepSpinner => true,
+          :url         => "management_console",
+          :klass       => ApplicationHelper::Button::GenericFeatureButton,
+          :options     => {:feature => :native_console}
```

Comment:
NOTE we're displaying this as "Management Console" on the Cloud VM page so keeping this consistent here, forget if we liked Management Console or if we wanted dashboard in there to differentiate from the VM's actual console @Fryguy 

We should do a follow-up to rename all of the methods and the feature to match what we're displaying on the UI

Reply:
I can't remember 😅  - Think we agreed on Management Console to go to the Provider (now that I wrote that maybe Provider console is better).

Reply:
That's good so no matter what we pick we'll like it, then hate it in a few weeks